package day0202;

public class UseRuntimeException {

	public static void main(String[] args) {

		int num;
		int num2;
		int result;

		try {
			num = Integer.parseInt(args[0]);
			num2 = Integer.parseInt(args[1]);

			try {
				result = num / num2;
				System.out.println(num + " / " + num2 + " = " + result);

			} catch (ArithmeticException ae) {

				System.out.println("나누는 값이 0이될 수 없음.");

			}

		} catch (NumberFormatException nfe) {

			System.out.println("인자값의 형식이 잘못되었음");
		}

		// main method의 arguments를 두개 입력받아 나눈 연산을 수행한 후
		// 결과를 보여주는 일.

	}// main

}// class
